invert_direction:
- false
- false
- false
- false
- false
- false
- false
- false
- false
- false
- false
- false
max_pulse_width:
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
min_pulse_width:
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
pulse_width_per_degree:
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
zero_degrees_pulse_width:
- 1509
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
